src/connector/src/sink/kafka.rs

```diff
@@ -290,9 +307,13 @@ impl Sink for KafkaSink {
         }
 
         // Try Kafka connection.
-        // TODO: Reuse the conductor instance we create during validation.
-        KafkaTransactionConductor::new(self.config.clone(), &"validation".to_string()).await?;
-
+        // There is no such interface for kafka producer to validate a connection
```

Comment:
May refactor the comment to be explicitly, e.g. "use enumerator to validate connection properties"